src/test/app/common/court-calculations/courtDecision.ts

```diff
@@ -43,11 +44,10 @@ describe('CourtDecision', () => {
       }).to.throw(Error, 'Input should be a moment, cannot be empty')
     })
 
-    it('should return a defendant decision type when no court date is supplied', () => {
-
-      let courtGeneratedPaymentDate = undefined
-
-      expect(CourtDecision.calculateDecision(null, null, courtGeneratedPaymentDate)).to.equal(DecisionType.DEFENDANT)
+    it('should return a defendant decision type when max future court date is supplied', () => {
```

Comment:
We are missing one important test, when court date is max future and claimant has made a more favourable offer than defendant. 

Or expressed in a different way, scenario [A] on the our court decision rules diagram should be tested with a court date that is max future and non max future.

Reply:
added a test for it, other scenarios are all covered